www/settings/app-settings.less

```diff
@@ -38,6 +38,22 @@
         margin-left: 0.5rem;
     }
 
+    .alertify {
+        nav {
```

Comment:
This affects all buttons navs in modals in settings. potentially some unintended consequences